Source/ASDisplayNode.mm

```diff
@@ -313,7 +319,16 @@ - (void)_staticInitialize
 - (void)_initializeInstance
 {
   [self _staticInitialize];
-  __instanceLock__.SetDebugNameWithObject(self);
+  if (ASNodeContext *ctx = ASNodeContextGet()) {
```

Comment:
Should we check here for the experiment activation instead of just creating one?

Reply:
This won't create one, it just checks it.